src/lang/java/mod.rs

```diff
@@ -20,13 +20,17 @@ pub fn merge_modules(modules: Vec<ModuleComponent>) -> Vec<ModuleComponent> {
     for module in modules.into_iter() {
         let name = module.module_name.clone();
         tracing::info!("Mod. Name: {}", name);
-        if packages.contains_key(&name) {
-            // tracing::info!("Merging...");
-            let orig_module = packages.get_mut(&name).expect("Contains key lied to me!");
-            orig_module.merge_into(module);
-        } else {
-            // tracing::info!("New! {}", module.module_name);
-            packages.insert(name, module);
+
+        match packages.entry(name) {
+            Entry::Occupied(mut packages) => {
```

Comment:
Thank goodness there was a solution to this, I hated checking the key, then having to unwrap anyway. It felt stupid.